02-intermediate-ruby/exception-handling.md

```diff
@@ -211,44 +174,36 @@ while true
 end
 ```
 
-### In Minitest
 
-If we expect a method to raise an exception and it doesn't, that's a bug! This means that we must test that our methods raise the exceptions we want, as well as testing nominal behavior. As we saw on in several projects, you can look for an exception in a given block of code in Minitest like this
+**Comprehension Questions:**
+
+Answer the comprehension prior to changing and executing the code to test your understanding.
 
+1. What happens if you remove the `=> exception` code?
+2. What happens if you change the `=> exception` code to `=> my_exception`?
+3. What happens if the `rescue` block is modified to include the code at the end of this loop?
 ```ruby
-it "Raises an ArgumentError when given an invalid word" do
```

Comment:
I do not understand question 3, or what is supposed to be done with the code. Could you clarify?

Reply:
Yes - I put the code below, but it wasn't clear the question.